src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -118,4 +120,41 @@ public OpenLoadFlowParameters setLowImpedanceBranchMode(LowImpedanceBranchMode l
     public List<AcLoadFlowObserver> getAdditionalObservers() {
         return additionalObservers;
     }
+
+    @AutoService(LoadFlowParameters.ConfigLoader.class)
+    public static class OpenLoadFlowConfigLoader implements LoadFlowParameters.ConfigLoader<OpenLoadFlowParameters> {
+
+        @Override
+        public OpenLoadFlowParameters load(PlatformConfig platformConfig) {
+            OpenLoadFlowParameters parameters = new OpenLoadFlowParameters();
+
+            platformConfig.getOptionalModuleConfig("open-loadflow-default-parameters")
+                    .ifPresent(config -> {
+                        parameters.setBalanceType(config.getEnumProperty("balanceType", BalanceType.class, BalanceType.PROPORTIONAL_TO_GENERATION_P_MAX));
```

Comment:
Maybe we could add a new class util/ParameterConstants.java in order to group all parameter name as :
balanceType, dc, distributedSlack, ...